build-scripts/versioning.cake

```diff
@@ -0,0 +1,140 @@
+using System.Text.RegularExpressions;
+
+public class BuildVersion
+{
+    private ISetupContext _context;
+    private BuildSystem _buildSystem;
+
+	public BuildVersion(ISetupContext context)
+	{
+        _context = context;
+        _buildSystem = context.BuildSystem();
+
+        PackageVersion = context.Argument("productVersion", "3.13.0");
```

Comment:
I think we should use the constant `DEFAULT_PRODUCT_VERSION` instead of `3.13.0`